rrfs-test/validated_yamls/templates/obtype_config/abi_g16.yaml

```diff
@@ -0,0 +1,488 @@
+       - filter: Variable Assignment
+         assignments:
+         - name: DerivedMetaData/surfaceFlag
+           type: int
+           function:
+             name: IntObsFunction/Conditional
+             options:
+               defaultvalue: 4
+               firstmatchingcase: true
+               cases:
+               - where:
+                 - variable:
+                     name: GeoVaLs/water_area_fraction
+                   minvalue: 0.99
+                 value: 0
+               - where:
+                 - variable:
+                     name: GeoVaLs/land_area_fraction
+                   minvalue: 0.99
+                 value: 1
+               - where:
+                 - variable:
+                     name: GeoVaLs/ice_area_fraction
+                   minvalue: 0.99
+                 value: 2
+               - where:
+                 - variable:
+                     name: GeoVaLs/surface_snow_area_fraction
+                   minvalue: 0.99
+                 value: 3
+
+       # Surface Parameter assignment from geovals
+       - filter: Variable Assignment
+         assignments:
+         - name: DerivedMetaData/surfaceParam
+           type: int
+           function:
+             name: IntObsFunction/Conditional
+             options:
+               defaultvalue: 0
+               firstmatchingcase: true
+               cases:
+               - where:
+                 - variable:
+                     name: GeoVaLs/water_area_fraction
+                   minvalue: 0.99
+                 value: 30
+               - where:
+                 - variable:
+                     name: GeoVaLs/land_area_fraction
+                   minvalue: 0.99
+                 value: 15
+               - where:
+                 - variable:
+                     name: GeoVaLs/ice_area_fraction
+                   minvalue: 0.99
+                 value: 20
+               - where:
+                 - variable:
+                     name: GeoVaLs/surface_snow_area_fraction
+                   minvalue: 0.99
+                 value: 15
+
+       # Criteria assignment for Thinning 
+       - filter: Variable Assignment
+         assignments:
+         - name: DerivedMetaData/thinningCriteria
+           type: int
+           function:
+             name: IntObsFunction/Arithmetic
+             options:
+               variables:
+               - name: DerivedMetaData/surfaceParam
+               coefs: [1]
+
+       # Thinning
+       - filter: Gaussian Thinning
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         horizontal_mesh: 60
+         use_reduced_horizontal_grid: true
+         distance_norm: maximum
+         time_mesh: 'PT06H'
+         time_min: '2024-05-26T22:30:00Z'
+         time_max: '2024-05-27T01:30:00Z'
+         priority_variable: DerivedMetaData/thinningCriteria
+         action:
+           name: reject
+
+       obs post filters:
```

Comment:
HI Xiaoyan, could you please double check if the following filters (e.g., online domain check, satellie zenith angle check, cloud fraction check, surface type check etc.) should be in the "obs prior filters" or should be in the "obs post filters"? To my understanding, it looks like for obs post filter, need to access hofx values.  So if no need hofx access, maybe no need to put them under post filters and maybe they should be in the prior filters (access Geovals) instead?  I am not very sure, what do you think? Thanks! @xyzemc